api/jobs.py

```diff
@@ -37,11 +38,33 @@
 def valid_transition(from_state, to_state):
     return (from_state + ' --> ' + to_state) in JOB_TRANSITIONS or from_state == to_state
 
-ALGORITHMS = [
-    'dcm2nii',
-    'qa'
+# Represet the general categories of gear function
+@unique
+class Category(Enum):
+    classifier  = 'classifier' # discover metadata
+    converter   = 'converter'  # translate between formats
+    qa          = 'qa'         # quality assurance
+    analytical  = 'analytical' # general purpose
+
+    # Enum strings are prefixed by their class: "Category.classifier".
+    # This overrides that behaviour and removes the prefix.
+    def __str__(self):
```

Comment:
Hmm, not a bad idea. I guess it depends more generally on how we'd like to use enums in the code base. 

The downside of that approach is that it wouldn't work if the enums aren't string-based, where the current approach would (if I used `self.name` in that function rather than `self.value`).

My eventual intention was that if we use enums more, we'd make a small helper class that does it for you, and let it handle the setup. Let me try that once and see how it feels.